src/krem.js

```diff
@@ -0,0 +1,7 @@
+export function krem(strings, ...args) {
+  const [namespace, id, ...rest] = args;
+  const styles = strings.slice(2).map((item, i) => {
+    return `${item}${rest[i] || ''}`;
+  }).join('');
+  return { styles, id, namespace };
```

Comment:
Ah I see -- this is a "postcss object" that `useCss` and `<Scoped>` already understand. Nice.